setup.py

```diff
@@ -48,12 +48,12 @@
         'wikipedia': ['rdflib==4.2.2'],
         'ir': ['faiss-cpu>=1.6.1', 'elasticsearch==7.5.1'],
         'spacy': ['spacy==2.3.0'],
-        'allennlp': ['allennlp==1.1.0', 'allennlp-models==1.1.0',
+        'allennlp': ['allennlp==1.2.0', 'allennlp-models==1.2.0',
                      'torch>=1.5.0'],
         'cliner': ['marisa-trie==0.7.4', 'scipy==1.5.0',
                    'scikit-learn==0.23.1', 'repoze.lru==0.7',
                    'tensorflow-gpu==1.12.0', 'python-crfsuite==0.9.7'],
-        'augment': ['elasticsearch==7.5.1']
+        'augment': ['elasticsearch==7.5.1', 'transformers>=3.1']
```

Comment:
Just to make sure, you didn't use `transformers`, this is from another PR?

Reply:
I used `transformers` in my tests for `MetaAugmentationWrapper` because their `BertForMaskedLM` is the intended augmentation model we use. But yes I did copy this from Yu Hao's back translation PR.

Reply:
I though we are using the Bert model from Texar.

Reply:
The classifier is from Texar but the aug model is from Transformer. Texar doesn't support it.

Reply:
Interesting to see Transformer and Texar work together well.